Extension/test/unitTests/updowngrade.test.ts

```diff
@@ -25,117 +22,156 @@ suite("UpgradeDowngrade", () => {
     const insider1: string = "0.27.1-insiders";
     const release0: string = "0.27.0";
 
-    suite("Downgrade", () => {
-        suite("Internal Testing, no Downgrade", () => {
-            test("Insider to Release", () => {
-                const builds: Build[] = [{
-                    name: release0, assets: four_assets}];
-
-                const userVersion: PackageVersion = new PackageVersion(insider1);
-                const targetBuild: Build | undefined = getTargetBuild(builds, userVersion, updateChannel);
-                assert.equal(targetBuild, undefined);
-            });
-
-            test("Insider to Insider", () => {
-                const builds: Build[] = [{
-                    name: insider2, assets: four_assets}, {
-                    name: insider1, assets: four_assets}, {
-                    name: release0, assets: four_assets}];
-
-                const userVersion: PackageVersion = new PackageVersion(insider3);
-                const targetBuild: Build | undefined = getTargetBuild(builds, userVersion, updateChannel);
-                assert.equal(targetBuild, undefined);
-            });
-        });
-        suite("Insider Users, Downgrade", () => {
-            test("Insider to Release", () => {
-                const builds: Build[] = [{
-                    name: insider1, assets: []}, {
-                    name: release0, assets: four_assets}];
-
-                const userVersion: PackageVersion = new PackageVersion(insider1);
-                const targetBuild: Build | undefined = getTargetBuild(builds, userVersion, updateChannel);
-                assert.equal(targetBuild.name, release0);
-            });
-
-            test("Insider to Insider", () => {
-                const builds: Build[] = [{
-                    name: insider2, assets: []}, {
-                    name: insider1, assets: four_assets}, {
-                    name: release0, assets: four_assets}];
-
-                const userVersion: PackageVersion = new PackageVersion(insider2);
-                const targetBuild: Build | undefined = getTargetBuild(builds, userVersion, updateChannel);
-                assert.equal(targetBuild.name, insider1);
-            });
-        });
-    });
-
-    suite("Upgrade", () => {
+    suite("DefaultChannel", () => {
+        const updateChannel: string = "Default";
+        const extra_insider2: string = "0.27.2-insiders2";
+        const extra_insider1: string = "0.27.2-insiders";
         suite("Automatic Upgrade", () => {
             test("Release to Release", () => {
```

Comment:
I may have unintentionally led you astray here. This function will only be called with "Default" as the release channel if you are exiting the "Insiders" program.  In this case the only action that should happen is "Automatic Downgrade" from "Insiders to Release".